src/components/UserProfile/ProfileUserActivity.tsx

```
@@ -55,8 +55,11 @@ const ProfileUserActivity = ({ className, userProfile }: Props) => {
 	const [filter, setFilter] = useState<EActivityFilter>(EActivityFilter.ALL);
 	const [totalCount, setTotalCount] = useState<number>(0);
 
+	console.log('userProfile', { userProfile, profileUserId });
```

ESLint

[error] Expected object keys to be in ascending order. 'profileUserId' should be before 'userProfile'.

(sort-keys)